shared/types/validator.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import pcrypto "github.com/pokt-network/pocket/shared/crypto"
+
+// The key is a hex encoded representation of the validator byte address.
+type ValMap map[string]*Validator
+
+type Validator struct {
```

Comment:
There's going to be a lot of contention around this structure as at this time there's not good justification to have a shared Validator type

It needs to be made more explicit that this is a temp